Arrowgene.Ddon.GameServer/ScheduleManager.cs

```diff
@@ -0,0 +1,89 @@
+using Arrowgene.Ddon.GameServer.Tasks;
+using Arrowgene.Ddon.Server;
+using Arrowgene.Ddon.Shared.Model.Scheduler;
+using Arrowgene.Logging;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Arrowgene.Ddon.GameServer
+{
+    public class ScheduleManager
+    {
+        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(ScheduleManager));
+
+        private List<SchedulerTask> Tasks;
+        private DdonGameServer Server;
+
+        private static readonly int TIMER_TICK_HOURLY = 1 * 1000; // 1 second
+        private static readonly int TIMER_TICK_DAILY = 10 * 1000; // 10 seconds
+        private static readonly int TIMER_TICK_WEEKLY = 30 * 1000; // 30 seconds
+
+        public ScheduleManager(DdonGameServer server)
+        {
+            Server = server;
+
+            // TODO: Load from server config
```

Comment:
I want to preserve across resets, that way say we have something like weekly dungeon rewards. I don't want important fixes to wait because we don't want players to be able to get `number of rewards * number of resets` when the rewards should be limited behind some timer.